### PR TITLE
test(e2e): add smoke test for use in CI builds

### PR DESCRIPTION
**Pull request by chriswldenyer**
For ER1-170.
This creates a 2 replica file-system volume of sufficient
size that I/O can run for at least 2 minutes, runs
fio to completion on the volume and deletes the pod
and the volume. The install and uninstall tests
implicitly test pool creation and deletion functionality.

### CODE REVIEW

File: scripts/test_lists.sh [hunk 1]
```diff
@@ -207,3 +207,7 @@ stale_msp_after_node_power_failure
 synchronous_replication
 volume_filesystem
 "
+
+profiles[ci_smoke]="
```
Comment (blaisedias) [inline scripts/test_lists.sh:211]: Apologies, test_lists.sh is not used anymore - I should have removed it.
Please add the test to `configurations/testlists.yaml`
there is already a section `smoke-test` which has `resource_check` replace that with this test.

**Review by blaisedias — CHANGES_REQUESTED** [review]

File: scripts/test_lists.sh (hunk 1 above)
Comment (chriswldenyer) [reply scripts/test_lists.sh]: Done

**Review by blaisedias — APPROVED** [review]
LGTM

**Review by blaisedias — APPROVED** [review]
LGTM

**Comment by chriswldenyer** [comment]
bors try

**Pull request merged**